src/GridFieldAddNewInlineButton.php

```diff
@@ -32,6 +32,8 @@ class GridFieldAddNewInlineButton extends AbstractGridFieldComponent implements
 
     private $title;
 
+    private $prependRows = false;
```

Comment:
```suggestion
    private bool $prependRows = false;
```
New API should have explicit type hints instead of PHPDocs.
If you choose to remove the configuration, ignore this.